### PR TITLE
updates to screenshots and docs link for codeship. Fixes #101

### LINKED ISSUE #101: Update Codeship screenshots and Documentation Link
Opening an issue to add some new screenshots, and update the README file for the Codeship example.  I will be working on this for you today, and submitting a PR.

### PR DESCRIPTION
**Pull request by kellyjandrews**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -853,22 +853,25 @@ Some initial set up of your project repo is required. This involves having a lam
...
 
-  ![Test script](https://cloud.githubusercontent.com/assets/5912647/12619098/83e071a6-c50e-11e5-8d28-45c8afc19f2e.png)
+  ![Setup Commands](https://user-images.githubusercontent.com/5222954/32383449-63517944-c08e-11e7-860f-aa7740806db3.png)
 
   You should already see the follow default code:
 
   ```bash
-  # By default we use the Node.js version set in your package.json or the latest
-  # version from the 0.10 release
+  # We support all major Node.js versions. Please see our documentation for a full list.
```
Comment (nelsonic) [inline README.md:867]: @kellyjandrews this refers to the Node.js Versions supported by AWS Lambda which is actually only `v4.3.2` and `6.10.3` see: http://docs.aws.amazon.com/lambda/latest/dg/current-supported-versions.html
therefore we deliberately select one of _these_ versions to run on CodeShip to ensure that if tests pass on CodeShip they will _actually_ run on Lambda. 😉

**Review by nelsonic — APPROVED** [review]
@kellyjandrews updated screenshots look good and copy changes make sense. thanks! 👍

**Comment by kellyjandrews** [comment]
Thank you!

File: README.md (hunk 1 above)
Comment (kellyjandrews) [reply README.md:867]: Right - I was updating the default text that shows up on the test script config page from Codeship, without changing your original tutorial.

**Pull request merged**